Alerts.html

```diff
@@ -0,0 +1,218 @@
+            margin-left: calc(5px * var(--scale));
+        }
+
+        .username {
+            text-transform: uppercase;
+            color: #FFA500;
+            font-size: calc(20px * var(--scale));
+            opacity: 0;
+        }
+
+        .message {
+            text-transform: uppercase;
+            font-size: calc(18px * var(--scale));
+            color: #ffffff;
+            margin-left: 0px;
+            opacity: 0;
+        }
+
+        .amount {
+            font-weight: 600;
+            font-family: 'Poppins', sans-serif;
+        }
+
+    </style>
+    <script
+        src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+    <script
+        src="https://cdnjs.cloudflare.com/ajax/libs/reconnecting-websocket/1.0.0/reconnecting-websocket.min.js"
+        integrity="sha512-B4skI5FiLurS86aioJx9VfozI1wjqrn6aTdJH+YQUmCZum/ZibPBTX55k5d9XM6EsKePDInkLVrN7vPmJxc1qA=="
+        crossorigin="anonymous"></script>
+    <script>
+        // Global Variables
+        let amount = 0;
+        let coinName;
+        // Initialise WebSocket Connection to Server
+        const socket = new WebSocket(
+            "wss://rally-alerts-server.herokuapp.com/rally/alerts/websocket"
+        );
+
+        // Reconnect on Close
+        /*socket.addEventListener("close", () => {
+          console.log("Reconnecting to Server");
+         setInterval(function () {
+            
+        });*/
+
+        socket.onclose = function () {
+            setInterval(function () {
+                new WebSocket(
+                    "wss://rally-alerts-server.herokuapp.com/rally/alerts/websocket"
+                );
+            }, 5000);
+        };
+
+        socket.onopen = function (event) {
+            
+        };
+        function isOpen(ws) {
+                return ws.readyState === ws.OPEN;
+            }
+            setInterval(function () {
+                if (!isOpen(socket)) {
+                    console.log("Reconnecting")
+                    location.reload();
+                    new WebSocket(
+                        "wss://rally-alerts-server.herokuapp.com/rally/alerts/websocket"
+                    );
+                }
+                else {
+                    socket.send("Keep Alive");
+                }
+            }, 5000);
+        // Listen to Data
+        socket.addEventListener("message", (event) => {
+            socket.send("Received!");
+            if (event.data.slice(0, 1) !== "{") {
+                console.log("SERVER: ", event.data);
+            } else {
+
+                if (JSON.parse(event.data).coinKind == coinName) {
+                    console.log("Donation Receieved");
+                    console.log(JSON.parse(event.data));
+                    Alert(event);
+                }
+            }
+        });
+
+        window.addEventListener("load", () => {
+            console.log(localStorage)
+            localStorage.setItem("coin", "SKOT");

```

Comment:
This is hardcoded to SKOT, is there a way to have time come from a config in OBS?

Reply:
Oh shoot, let me get that corrected

Reply:
Now that I see why I did that, the default coin is set to SKOT, which can then be changed through the activity feed settings. It was hardcoded just so there could be a default coin in there and have it not be empty